cupy/testing/helper.py

```diff
@@ -173,8 +175,21 @@ def numpy_cupy_allclose(rtol=1e-7, atol=0, err_msg='', verbose=True,
     .. seealso:: :func:`cupy.testing.assert_allclose`
     """
     def check_func(cupy_result, numpy_result):
-        array.assert_allclose(cupy_result, numpy_result,
-                              rtol, atol, err_msg, verbose)
+        c = cupy_result
+        n = numpy_result
+        array.assert_allclose(c, n, rtol, atol, err_msg, verbose)
+        if (contiguous_check and
+                isinstance(c, cupy.ndarray) and isinstance(n, numpy.ndarray)):
```

Comment:
Why do you have to check types here?
If we need to check it, shouldn't it be checked regardless of `contiguous_check`?

Reply:
This branch supports single value or list.

Reply:
Could you elaborate?
Do you mean `c` and `n` could be lists?

Reply:
Yes. When we use `parameterize` test, sometimes NumPy returns not ndarray type.